### ISSUE TITLE
Docs request for combination with vendor chunks and or AutoDLL Plugin

### ISSUE THREAD

**Issue by swernerx** [faceyspacey/webpack-flush-chunks#18]
I am currently trying to add the AutoDll Plugin to my build chain as it was reported before to work with this solution. Unfortunately I had no direct success. The vendor entry point does not seem to be visible for this plugin. It just lists the default set of bootstrap + dynamic + main. But no vendor. Is there any magic trick or option to utilize?

**Comment by swernerx** [comment]
I was wondering whether this is related to: https://github.com/asfktz/autodll-webpack-plugin/issues/23

**Comment by faceyspacey** [comment]
It is, and I researched it to attempt to do a PR to the autodll package--basically Webpack has it planned for the future to allow plugin authors to add to stats. Currently we cannot--unless I'm missing something. 

So the chunk created by autodll won't be `stats.assetsByChunkName`, and there's nothing we can do about it. You just gotta append it to the page manually like we always have done. 

Feel free to add it to the readme as a PR.

The challenge with the setup I've proposed through these tools is that no matter which way you slice it, it requires lots of setup/configuration. That's the price we pay for *frameworkless* flexibility vs. something like Next.js.

Basically, the notes on webpack "externals" is the beginning of a `troubleshooting.md`. I'm not sure if the entire webpack configuration docs should go there, but probably things like the hashed/named module ids stuff should go there as well. We're gonna need a complete list of "gotchas" and configuration requirements in one place. 

Webpack Flush Chunks fyi will be renamed universal-render. I already own the npm package. There's a lot more coming out. Any help is much appreciated to bring this all together.
